t/Jira.t

```diff
@@ -4,17 +4,40 @@ use strict;
 use warnings;
 use utf8;
 
-BEGIN { unshift(@INC, $ENV{'SDRROOT'}. '/crms/cgi'); }
+use lib "$ENV{SDRROOT}/crms/lib";
 
-use Encode;
+use Test::LWP::UserAgent;
 use Test::More;
 
-use Jira;
+use CRMS::Config;
+use CRMS::Jira;
 
-is(Jira::LinkToJira('HT-000'),
-   '<a href="https://hathitrust.atlassian.net/browse/HT-000" target="_blank">HT-000</a>',
-   'Jira::LinkToJira produces the correct URL');
-my $req = Jira::Request('GET', 'some/path/to/something');
-isa_ok $req, "HTTP::Request";
+my $jira = CRMS::Jira->new;
+
+subtest '#add_comment' => sub {
+  my $prefix = CRMS::Config->new->config->{'jira_prefix'};
+  my $ua = Test::LWP::UserAgent->new;
+  $ua->map_response(qr{000}, HTTP::Response->new('200', 'OK', ['Content-Type' => 'text/plain'], ''));
```

Comment:
This took me a minute to figure out - I think it's just looking for anything with "000" in the URL?

Reply:
Which is to say, it might be worth specifically just mapping the endpoints we care about, or if that seems too brittle, at least including the project key and/or making it a little more explicit, e.g. `qr{^.*http://.*/HT-000$}`